json-schema/schema.json

```diff
@@ -464,7 +464,10 @@
         "torch.export.save",
         "tf.keras.Model.save",
         "tf.keras.Model.save_weights",
-        "tf.saved_model.export(format='tf_saved_model')"
+        "tf.keras.Model.export",
+        "safetensors.torch.save_file",
+        "safetensors.tensorflow.save_file",
+        "safetensors.paddle.save_file"
```

Comment:
should we add flax here as well and as an option in best-practices? Would be good to have some more jax/flax representation but not urgent if you'd just like to merge this addition. 

https://huggingface.co/docs/safetensors/api/flax

https://github.com/google/flax

Reply:
done